data/web/inc/vars.inc.php

```diff
@@ -42,11 +42,13 @@
 $MAILCOW_APPS = array(
   array(
     'name' => 'SOGo',
-    'link' => '/SOGo/'
+    'link' => '/SOGo/',
+    'description' => 'SOGo is a collaborative software (groupware) server.'
```

Comment:
@mkuron how about this one "SOGo is a multi-purpose groupware server that provides native Microsoft ActiveSync support."? 😄